From bb43e6ee8d6415cbdc9ffc7e37c5022133bb50da Mon Sep 17 00:00:00 2001
From: "Kirill A. Korinsky" <kirill@korins.ky>
Date: Sun, 20 Feb 2022 22:19:43 +0100
Subject: [PATCH] Darwin, libgcc: include support macOS 12+

This issue was introduced at previous commit:
```
Author: Iain Sandoe <iain@sandoe.co.uk>
Date:   Sun Jan 3 12:24:45 2021 +0000

    Darwin, libgcc : Adjust min version supported for the OS.
```

Signed-off-by: Kirill A. Korinsky <kirill@korins.ky>
---
 libgcc/config.host | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git libgcc/config.host libgcc/config.host
index 953fc3abceb..4482333d67d 100644
--- libgcc/config.host
+++ libgcc/config.host
@@ -249,7 +249,7 @@ case ${host} in
     echo "enable_darwin_at_rpath is '$enable_darwin_at_rpath'" 1>&2
   fi
   case ${host} in
-    *-*-darwin20* | *-*-darwin1[89]*)
+    *-*-darwin2* | *-*-darwin1[89]*)
       tmake_file="t-darwin-min-8 $tmake_file"
       ;;
     *-*-darwin9* | *-*-darwin1[0-7]*)
-- 
2.40.1

